src/Seabug/Contract/Util.purs

```diff
@@ -0,0 +1,52 @@
+module Seabug.Contract.Util
+  ( minAdaOnlyUTxOValue
+  , setSeabugMetadata
+  ) where
+
+import Contract.Prelude
+
+import Contract.AuxiliaryData (setTxMetadata)
+import Contract.Monad (Contract, liftContractM)
+import Contract.Numeric.Natural (toBigInt)
+import Contract.ScriptLookups (UnattachedUnbalancedTx)
+import Contract.Value (CurrencySymbol)
+import Contract.Value as Value
+import Data.BigInt (BigInt)
+import Data.BigInt as BigInt
+import Seabug.Metadata.Share (mkShare)
+import Seabug.Metadata.Types (SeabugMetadata(..))
+import Seabug.Types (NftData(..))
+
+minAdaOnlyUTxOValue :: BigInt
+minAdaOnlyUTxOValue = BigInt.fromInt 1_000_000
+
+-- | Set metadata on the transaction for the given NFT
+setSeabugMetadata
+  :: forall (r :: Row Type)
+   . NftData
+  -> CurrencySymbol -- | The currency symbol of the self-governed nft
+  -> UnattachedUnbalancedTx
+  -> Contract r UnattachedUnbalancedTx
+setSeabugMetadata (NftData nftData) sgNftCurr tx = do
+  let
+    nftCollection = unwrap nftData.nftCollection
+    nftId = unwrap nftData.nftId
+    natToShare nat = liftContractM "Invalid share"
+      $ mkShare
+      =<< BigInt.toInt (toBigInt nat)
+    policyId = Value.currencyMPSHash sgNftCurr
+  authorShareValidated <- natToShare nftCollection.authorShare
+  marketplaceShareValidated <- natToShare nftCollection.daoShare
+  setTxMetadata tx $ SeabugMetadata
+    { policyId
+    , mintPolicy: mempty
```

Comment:
#13

Reply:
I guess we can just hardcode something like `"V1"` for now? Not sure what the best solution is in the long term, maybe some kind of environment file and passing config from there, like we're doing with `projectId`